### DISCUSSION TITLE
Conflict errors

### DISCUSSION THREAD

**Discussion by superdev223** [community/community#58110] [Accessibility]
### Select Topic Area

Question

### Body

During merge, I got conflict error. How can I solve it?

**Comment by HappyMicky0317** [comment]
To solve a conflict error during a merge, you can follow these steps:

Identify the conflicting files and open them in your preferred code editor.
Look for the conflicting lines of code and determine which changes you want to keep.
Edit the files to resolve the conflicts and save your changes.
Add the modified files to the staging area using the git add command.
Commit the changes using the git commit command with a message that describes the changes you made to resolve the conflict.
If you need more specific help with your merge conflict, please provide more details about your situation.

**Reply by superdev223** [reply]
This is helpful. Thanks happydream0317

**Reply by queenofcorgis** [reply]
@superdev223 @HappyDream0317 GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
